02-getting_started.Rmd

```diff
@@ -1,4 +1,4 @@
-# Getting Started 
+# Getting Started {#GetStart}

```

Comment:
I would prefer a better plain text shortener here... IINM these anchors are not just for referring internally in the page -- it can show up eventually in a URL as URL/getting-started#GetStart; something more readable would be better. I may be mistaken but I think this may generate an automatic anchor like `#getting-started`? If not that would anyway be a good choice.